<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter_37">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QCustomPlot" name="customPlot" native="true">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter_36">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_35">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QDial" name="Angle"/>
         <widget class="QSlider" name="Speed">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_26">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>ID</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="IDEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>E</string>
           </property>
          </widget>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>更新参数</string>
          </property>
         </widget>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="minimumSize">
          <size>
           <width>291</width>
           <height>126</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>291</width>
           <height>126</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QSplitter" name="splitter_15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>电压 </string>
              </property>
             </widget>
             <widget class="QLineEdit" name="voltline">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_14">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>速度 </string>
              </property>
             </widget>
             <widget class="QLineEdit" name="speedline">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSplitter" name="splitter_16">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label_3">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>温度 </string>
              </property>
             </widget>
             <widget class="QLineEdit" name="temline"/>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSplitter" name="splitter_17">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QLabel" name="label_4">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>位置 </string>
              </property>
             </widget>
             <widget class="QLineEdit" name="posiline"/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_13">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QSplitter" name="splitter_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QLabel" name="label_6">
               <property name="minimumSize">
                <size>
                 <width>43</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>速度P</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_IP">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>I</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_II">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>D</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_ID">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </widget>
             <widget class="QSplitter" name="splitter_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QLabel" name="label_7">
               <property name="minimumSize">
                <size>
                 <width>43</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>位置P</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_PP">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>I</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_PI">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>D</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_PD">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </widget>
             <widget class="QSplitter" name="splitter_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QLabel" name="label_8">
               <property name="minimumSize">
                <size>
                 <width>43</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>弱磁P</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_AP">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>I</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_AI">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>D</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEdit_AD">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QCheckBox" name="SpeedCheck">
           <property name="text">
            <string>速度模式</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="AngleCheck">
           <property name="text">
            <string>角度模式</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QComboBox" name="Serial">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
          </widget>
          <widget class="QPushButton" name="Open">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </widget>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QLineEdit" name="Send_edit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="Reset">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>  重置  </string>
          </property>
         </widget>
         <widget class="QPushButton" name="Send">
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>发送指令</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
